tests/unit/local/docker/test_container.py

```diff
@@ -311,6 +311,72 @@ def test_must_fail_if_already_created(self):
             container.create()
 
 
+class TestContainer_stop(TestCase):
+    def setUp(self):
+        self.image = IMAGE
+        self.cmd = "cmd"
+        self.working_dir = "working_dir"
+        self.host_dir = "host_dir"
+
+        self.mock_docker_client = Mock()
+        self.mock_docker_client.containers = Mock()
+        self.mock_docker_client.containers.get = Mock()
+
+        self.container = Container(
+            self.image, self.cmd, self.working_dir, self.host_dir, docker_client=self.mock_docker_client
+        )
+        self.container.id = "someid"
+
+        self.container.is_created = Mock()
+
+    def test_stop_with_timeout(self):
+        self.container.is_created.return_value = True
+        real_container_mock = Mock()
+        self.mock_docker_client.containers.get.return_value = real_container_mock
+        real_container_mock.remove = Mock()
+
+        self.container.stop(time=3)
```

Comment:
`3` is the default value, you should maybe use another one

Reply:
This was mainly an explicit over implicit thing, in case it's changed by the Docker SDK (which has made breaking changes in the past). Will keep this way, but fair callout.